personalsite/src/components/api.js

```diff
@@ -0,0 +1,49 @@
+import '../App.css'; 
+import axios from 'axios'; 
+import { Component } from 'react'; 
+
+class TriviaAPI extends Component { 
+    constructor(props) { 
+        super(props); 
+        this.state = { 
+            data: null
+        }
+    }
+    //async func 
+    fetchData = () => { 
+        axios.get("http://jservice.io/api/random")
+        .then((response) => { 
+            console.log(response.data); 
+            this.setState({data: response.data[0]})
+        }).catch((error) => { 
+            console.log(error)
+        })
+    }
+
+    renderData = () => { 
+        if(this.state.data) { 
+            return( 
+                <div> 
+                    <em>Clue: </em> <br />
+                    {this.state.data.question}
+                    <br/> 
+                    {/* <input type="text">Your Answer: </input> */}
```

Comment:
I like where you were going with this. You could have a function that saves the input to a state variable and then a "check" button that has a function that compares what you had in the input box to the answer of the question which is saved in `this.state.data`!